manifests/vizier/db/deployment.yaml

```diff
@@ -31,6 +31,15 @@ spec:
         ports:
         - name: dbapi
           containerPort: 3306
+        readinessProbe:
```

Comment:
It would be nice if vizier-core can get readinessprobe or readinessgate, too.

Reply:
I think so too, will do in another PR maybe.

Reply:
Yup. We can do it later.

Reply:
Opened an issue: https://github.com/kubeflow/katib/issues/268